aws/resource_aws_route53_health_check_test.go

```diff
@@ -185,7 +194,39 @@ func TestAccAWSRoute53HealthCheck_IpConfig(t *testing.T) {
 	})
 }
 
+func TestAccAWSRoute53HealthCheck_Ipv6Config(t *testing.T) {
+	var check route53.HealthCheck
+	resourceName := "aws_route53_health_check.test"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckRoute53HealthCheckDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccRoute53HealthCheckIpv6Config,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckRoute53HealthCheckExists(resourceName, &check),
+					resource.TestCheckResourceAttr(resourceName, "ip_address", "1234:5678:9abc:6811:0:0:0:4"),
+				),
+			},
+			{
```

Comment:
To make the tests explicit, you could add some `plan`-only tests, e.g.
```go
{
  Config:   testAccRoute53HealthCheckIpv6ExpandedConfig,
  PlanOnly: true,
}
```

Reply:
changed and re ran test:

--- PASS: TestAccAWSRoute53HealthCheck_Ipv6Config (59.54s)